webui/react/src/components/SelectFilter.tsx

```diff
@@ -77,20 +77,13 @@ const SelectFilter: React.FC<Props> = forwardRef(function SelectFilter(
     setIsOpen(open);
   }, []);
 
-  // eslint-disable-next-line  @typescript-eslint/no-explicit-any
-  const handleFilter = useCallback((search: string, option: any) => {
-    /*
-     * `option.children` is one of the following:
-     * - undefined
-     * - string
-     * - string[]
-     */
+  const handleFilter = useCallback((search: string, option?: DefaultOptionType): boolean => {
```

Comment:
added a concrete type instead of `any`

`webui/react/node_modules/rc-select/lib/Select.d.ts`
<img width="539" alt="image" src="https://user-images.githubusercontent.com/109113616/217377471-870cf103-3f35-46cd-83de-7e830c53fccf.png">